versioned_docs/version-2.x/advanced/analytics/plugins.mdx

```diff
@@ -344,6 +344,12 @@ load / remove (depending on the `userConsents` value). Here is an example:
 
 </Figure>
 
+#### Retrieve authorized cookie services in GTM's datalayer
+
+If your project needs to retrieve the authorized cookies services in GTM's
+datalayer, in example for triggering other analytics services, you can do so by
+[declaring them in your `analytics.js`](./getting-started#retrieve-authorized-cookie-services-in-analytics).
```

Comment:
**nitpick:** it feels like a complement to the previous explanation

**suggestion:** 
```suggestion
Please note that to retrieve the authorized cookies services in GTM's
datalayer, services must be [declared in your `analytics.js`](./getting-started#retrieve-authorized-cookie-services-in-analytics).
```